hibernate-envers/src/test/java/org/hibernate/envers/test/embeddedid/TestRunner.java

```diff
@@ -0,0 +1,83 @@
+package org.hibernate.envers.test.embeddedid;
+
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.boot.MetadataSources;
+import org.hibernate.boot.registry.StandardServiceRegistry;
+import org.hibernate.boot.registry.StandardServiceRegistryBuilder;
+import org.hibernate.envers.test.BaseEnversFunctionalTestCase;
+import org.hibernate.envers.test.Priority;
+import org.hibernate.testing.TestForIssue;
+import org.junit.Before;
+import org.junit.Test;
+
+@TestForIssue(jiraKey = "HHH-13361")
+public class TestRunner extends BaseEnversFunctionalTestCase {

```

Comment:
Please rename this test to something like `NestedEmbeddedIdTest`.  It helps better identify what the test is actually trying to accomplish at a glance rather than having to read the code, comments, or the jira to gather that.